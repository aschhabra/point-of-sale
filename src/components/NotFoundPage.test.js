
import React from "react";
import {shallow} from "enzyme";
import NotFoundPage from "./NotFoundPage";

test( "test should render Not Found with expenses",()=>{
  const wrapper= shallow(<NotFoundPage/>);


  expect(wrapper).toMatchSnapshot();
});
