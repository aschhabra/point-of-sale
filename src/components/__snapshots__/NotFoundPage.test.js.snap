// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test should render Not Found with expenses 1`] = `
<div>
  404 ! - 
  <Link
    replace={false}
    to="/"
  >
    Going home 
  </Link>
</div>
`;
